docs/source/concept_models/vivarium_ciff_sam/concept_model.rst

```diff
@@ -167,7 +222,7 @@ Scale up the coverage of treatment and prevention interventions to 90%.
 5.1 Vivarium concept model diagram
 ----------------------------------
 
-.. image:: sam_concept_model_diagram.svg
+.. image:: am_concept_model_diagram.svg
```

Comment:
Should we still have a correlation (dashed line) between wasting and stunting in this diagram even though we took the causal arrow out?

Reply:
Yeah... I think we'll have to think through if it makes more sense to correlate stunting with birth weight or stunting with with wasting directly. I can think through this with the rest of the risk risk correlation documentation!

Reply:
But either way, the three of those risks will be correlated together somehow, which is implied through the current concept model, so I think its not super critical to add a dashed line between wasting and stunting, although we could to be explicit!